packages/fuels/src/bin.ts

```diff
@@ -1,12 +1,7 @@
 import { error } from './cli/utils/logger';
 import { run } from './run';
 
-try {
-  run(process.argv).catch((x) => {
-    // eslint-disable-next-line no-console
-    console.log(x);
-  });
-} catch (err: unknown) {
+run(process.argv).catch((err) => {
```

Comment:
This `catch` was previously swallowing the error and the `catch(err: unknown)` part of the `try/catch` would never be called and thus `process.exit(1)` was never called.